migrations/2023-08-04-100313_login-token/up.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE login_token (
+    id serial PRIMARY KEY,
+    token text NOT NULL,
```

Comment:
Instead of storing the full token string, it should be enough to store the `iat` (token creation timestamp). Or put a short random string in `jti` and store that.

https://datatracker.ietf.org/doc/html/rfc7519#section-4.1